vms/platformvm/txs/convert_subnet_tx.go

```diff
@@ -62,3 +87,24 @@ func (tx *ConvertSubnetTx) SyntacticVerify(ctx *snow.Context) error {
 func (tx *ConvertSubnetTx) Visit(visitor Visitor) error {
 	return visitor.ConvertSubnetTx(tx)
 }
+
+type ConvertSubnetValidator struct {
+	// TODO: Must be Ed25519 NodeID
+	NodeID ids.NodeID `serialize:"true" json:"nodeID"`
+	// Weight of this validator used when sampling
+	Weight uint64 `serialize:"true" json:"weight"`
+	// Initial balance for this validator
+	Balance uint64 `serialize:"true" json:"balance"`
+	// [Signer] is the BLS key for this validator.
+	// Note: We do not enforce that the BLS key is unique across all validators.
+	//       This means that validators can share a key if they so choose.
+	//       However, a NodeID + Subnet does uniquely map to a BLS key
+	Signer signer.Signer `serialize:"true" json:"signer"`
```

Comment:
Rather than using `signer.Signer` - should we just us the `bls.ProofOfPossession` struct? If we were to ever change the format of the BLS key... having the interface may be nice... But it feels a bit weird to use an interface where only one value is valid.